scripts/cnode-helper-scripts/env

```diff
@@ -389,7 +376,20 @@ getAnswerAny() {
 #             : 2 = update failed
 checkUpdate() {
   [[ "${UPDATE_CHECK}" != "Y" ]] && return 0
-  if curl -s -f -m 10 -o "${PARENT}/${1}".tmp "${URL}/${1}" 2>/dev/null; then
+
+  if [[ ${OFFLINE_MODE} = N && ${BRANCH} != master && ${BRANCH} != alpha ]]; then
+    if ! curl -s -f -m ${CURL_TIMEOUT} "https://api.github.com/repos/cardano-community/guild-operators/branches" | jq -e ".[] | select(.name | contains(\"${BRANCH}\"))" &>/dev/null ; then
```

Comment:
Works for most cases, just an edge case here - `cntools-8.4.1` vs `cntools-8.4.11` might not get caught

Reply:
correct, better to do exact match, fixing.